rudra/utils/github_helper.sh

```diff
@@ -39,6 +39,10 @@ file_name=${git_home}/bay-services/$1
 branch_name=bump-$1-$3
 model_version=$3
 
```

Comment:
PR creation shall be simplified using `gh` cli tool. Anyways, that can be separate activity.
^^ @sawood14012

Reply:
Ack

Reply:
Raised a separate ticket for this at https://issues.redhat.com/browse/APPAI-1734